dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5505,6 +5506,24 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_MEMBERSHIP_MANAGER_TYPE =
+      enumBuilder(Name.WORKER_MEMBERSHIP_MANAGER_TYPE, MembershipType.class)
+          .setDefaultValue(MembershipType.NOOP.name())
+          .setDescription("Type of membership manager used for workers."
+              + "Choose STATIC for pre-configured members."
+              + "Choose ETCD for using etcd for membership management")
```

Comment:
add a line in desc on NOOP (for our engineers)

Reply:
done